requirements.txt

```diff
@@ -1,2 +1,2 @@
-git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
+git+https://github.com/edx/xblock-utils.git@v1.0.4#egg=xblock-utils==1.0.4
```

Comment:
So instead of the update I made here: https://github.com/edx/xblock-utils/commit/890bac00bca0df85c92a12c198dbb071b89e7b8d

would you have changed the `==` to a `>=`?